zeppelin-web/src/components/navbar/navbar.html

```diff
@@ -62,15 +62,17 @@
         <li ng-if="ticket">
         <!--TODO(bzz): move to Typeahead https://angular-ui.github.io/bootstrap  -->
           <form role="search"
-            style="width: 300px; display: inline-block; margin: 0 10px"
+            style="display: inline-block; margin: 0 10px"
+             class="navbar-form"
              ng-submit="search(searchTerm)">
             <div class="input-group">
               <input
                 type="text"
                 ng-model="searchTerm"
                 ng-disabled="!navbar.connected"
                 class="form-control"
-                placeholder="Search in your notebooks"
+                tooltip-placement="bottom"
+                tooltip="Search for your Notebooks"
```

Comment:
@minahlee @AhyoungRyu Thank you for you are feedback.
Reasons are as follows. Reducing the width of the search box will show all the text in the placeholder. Since you already have the search icon, you can not misunderstand the function of the search box. For detailed function descriptions are replaced by the tooltip.